holidays/countries/israel.py

```diff
@@ -83,42 +101,42 @@ def _populate_optional_holidays(self):
         if self._year >= 2101:
             raise NotImplementedError
 
-        # Sukkot Chol HaMoed
-        sukkot_dt = self._get_holiday("SUKKOT")
+        sukkot_dt = self._get_holiday(SUKKOT)
         for offset in range(1, 6):
-            self._add_holiday("Sukkot", sukkot_dt + td(days=offset))
+            # Chol HaMoed Sukkot (Feast of Tabernacles holiday).
+            self._add_holiday(tr("חול המועד סוכות"), sukkot_dt + td(days=offset))
 
         if self._year >= 2008:
-            # Sigd
-            self._add_holiday("Sigd", self._get_holiday("YOM_KIPPUR") + td(days=+49))
+            # Sigd.
+            self._add_holiday(tr("סיגד"), self._get_holiday(YOM_KIPPUR) + td(days=+49))
 
-        # Purim
-        self._add_holiday("Purim", self._get_holiday("PURIM"))
+        # Purim.
+        self._add_holiday(tr("פורים"), self._get_holiday(PURIM))
 
-        # Passover (Chol HaMoed Pesach)
-        passover_dt = self._get_holiday("PASSOVER")
+        passover_dt = self._get_holiday(PASSOVER)
         for offset in range(1, 6):
-            self._add_holiday("Passover", passover_dt + td(days=offset))
+            # Chol HaMoed Pesach (Passover holiday).
+            self._add_holiday(tr("חול המועד פסח"), passover_dt + td(days=offset))
 
-        # Memorial Day
         if self._year >= 1963:
-            name = "Memorial Day"
-            memorial_day_dt = self._get_holiday("INDEPENDENCE_DAY") + td(days=-1)
+            # Yom Hazikaron (Fallen Soldiers and Victims of Terrorism Remembrance Day).
+            name = tr("יום הזיכרון לחללי מערכות ישראל ונפגעי פעולות האיבה")
+            remembrance_day_dt = self._get_holiday(INDEPENDENCE_DAY) + td(days=-1)
             rule = THU_TO_PREV_WED + FRI_TO_PREV_WED
             if self._year >= 2004:
                 rule += SUN_TO_NEXT_MON
-            is_obs, _ = self._add_observed(memorial_day_dt, name, rule)
-            if not is_obs:
-                self._add_holiday(name, memorial_day_dt)
+            is_observed, _ = self._add_observed(remembrance_day_dt, name, rule)
+            if not is_observed:
+                self._add_holiday(name, remembrance_day_dt)
 
-        # Jerusalem Day
-        self._add_holiday("Jerusalem Day", self._get_holiday("LAG_BAOMER") + td(days=+10))
+        # Yom Yerushalayim (Jerusalem Day).
+        self._add_holiday(tr("יום ירושלים"), self._get_holiday(LAG_BAOMER) + td(days=+10))
```

Comment:
this would not have been celebrated before 1967

Reply:
Done